<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnswerDialog</class>
 <widget class="QDialog" name="AnswerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #A4A8A6;</string>
  </property>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-size : 15px;
background-color : #05BF11;
color: &quot;white&quot;;
font-weight: bold;
border: 1px solid;
border-radius: 4px;
border-color: #95FF93;
}

 QPushButton:hover
{
background-color : green;
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_answer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 15px;
color: #0048FF;

font-weight: bold;
border-color: #95FF93;
background-color: #e9e9e9;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>-120</x>
     <y>0</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-weight: bold;
font-size: 18px;
border: 1px solid;
background-color: #06A310;</string>
   </property>
   <property name="text">
    <string>A N S W E R</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
